tests/jobs/scala/src/main/scala/KafkaCassandraStream.scala

```diff
@@ -0,0 +1,385 @@
+import java.util.{Date, Properties}
+import java.util.UUID.randomUUID
+
+import com.datastax.spark.connector.cql.CassandraConnector
+import org.apache.kafka.clients.producer.{KafkaProducer, ProducerConfig, ProducerRecord}
+import org.apache.spark.mllib.random.RandomRDDs.normalRDD
+import org.apache.spark.sql.SparkSession
+import org.apache.spark.streaming.dstream.ConstantInputDStream
+import org.apache.spark.streaming.{Seconds, StreamingContext}
+import scopt.OptionParser
+
+import scala.math.max
+import scala.util.Random
+import com.datastax.spark.connector._
+import org.apache.kafka.clients.consumer.ConsumerConfig
+import org.apache.kafka.common.serialization.StringDeserializer
+import org.apache.spark.internal.Logging
+import org.apache.log4j.{Level, Logger}
+import org.apache.spark.streaming.kafka010.{ConsumerStrategies, KafkaUtils, LocationStrategies}
+
+case class Employee(
+                     id: String,
+                     first_name: String,
+                     last_name: String,
+                     email_address: String,
+                     address: String,
+                     city: String
+                   )
+
+/**
+  * A config glass used to set the command line arguments.
+  */
+case class Config(
+                   appName: String = "KafkaRandomFeeder",
+                   brokers: String = "" ,
+                   topics: String = "",
+                   groupId: String = "TODO_USE_A_DIFFERENT_GROUP_ID",
```

Comment:
Just a heads up, if you use the same hardcoded groupId for all consumers, then you may end up in a situation where consumers read from the queue one a time while all other consumers sit idle. I found that in my scale testing for PR #302

Reply:
Yes, I saw that comment. Thanks for the reminder.